modules/kn-service-apply.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * serverless/knative_serving/serverless-applications.adoc
+// * serverless/cli_reference/kn-services-ref.adoc
+
+[id="kn-service-apply_{context}"]
+= Applying service declarations
+
+You can apply declarations to a Knative service by using the `kn` CLI `kn service apply` command.
+
+.Example commands
+
+* Create a service:
++
+[source,terminal]
+----
+$ kn service apply <service-name> --image <image>
+----
+
+* Add an environment variable to a service:
++
+[source,terminal]
+----
+$ kn service apply <service-name> --image <image> --env <key>=<value>
+----
++
+[NOTE]
+====
+All parameters must be fully specified when using `kn service apply` to add an environment variable.
```

Comment:
That is true not only when adding an environment. In general, you always have to provide the full configuration with every apply command (so its better suited for a scripted context or CI/CD pipelines than for interactive and incremental update steps). But see comment above, we could add some more content to the intro.

Reply:
I think we can remove this note now since your suggestion for the intro bit above clarifies this better 😄